### ISSUE TITLE
[FEATURE] Change Action Button Colour Scheme

### ISSUE THREAD

**Issue by rmdtech** [rmdtech/ASE-Summ2#64]
### Description 📝
Google Lighthouse Accessibility Report has highlighted a contrast ratio issue in relation to the application's action buttons. To resolve this, the text colour should be changed to improve the contrast ratio. 

### Objectives 🎯
- [ ] Change the text colour on application buttons.
- [ ] Definition of Done: Text colour is changed to #12273F

### Dependencies ➕
N/A

### Notes 📌
Further research shows that the original design utilised the incorrect colour on UI buttons, these should be changed to dark blue to better utilise the colour scheme provided.
